src/main/scala/it/pagopa/interop/backendforfrontend/error/BFFErrors.scala

```diff
@@ -134,4 +134,7 @@ object BFFErrors {
 
   final case class InstitutionNotFound(selfcareId: UUID)
       extends ComponentError("0033", s"Institution $selfcareId not found")
+
+  final case class UnknownTenantSubUnitType(subUnitType: String)
+      extends ComponentError("0034", s"Tenant Sub Unit Type ${subUnitType} is not inside whitelist")
```

Comment:
The API validator already checks that the correct enum is passed (and if required returns 400), so there is no need for a specific error class

```suggestion
```